libraries/aws_iam_policies.rb

```diff
@@ -30,7 +30,6 @@ def initialize(opts = {})
     parameters = {}
     parameters[:only_attached] = opts[:only_attached] if opts[:only_attached]
     parameters[:scope]         = opts[:scope]         if opts[:scope]
-    parameters[:path_prefix]   = opts[:path_prefix]   if opts[:path_prefix]
```

Comment:
Does this removal break any ITs?

Reply:
Nope. I forgot to remove it in a previous PR. Had considered adding it in, but decided against it.